src/main/java/com/desiato/puresynth/PureSynthApplication.java

```diff
@@ -1,14 +1,18 @@
 package com.desiato.puresynth;
 
+
```

Comment:
To be purist, this PR should not contain new lines in this file.

Reply:
yes it makes sense :)